test/models/vote_test.rb

```diff
@@ -0,0 +1,120 @@
+require "test_helper"
+
+describe Vote do 
+  before do
+    @user = users(:leah)
+    @work = works(:moonstruck)
+    @vote = Vote.create(user_id: @user.id, work_id: @work.id)
```

Comment:
Instead of creating a vote here, you could create a fixture for this vote. Fixtures are a great place to store valid data that you want to test.